dear-imgui.cabal

```diff
@@ -94,6 +94,15 @@ flag sdl
   manual:
     True
 
+flag sdl-renderer
+  description:
+    Enable SDL Renderer backend (requires the SDL_RenderGeometry function available in SDL 2.0.18+).
```

Comment:
The ImGui preprocessor error calls for 2.0.17+, but I opted for 2.0.18+ here as that's when the official [docs](https://wiki.libsdl.org/SDL2/SDL_RenderGeometry#version) and [release notes](https://github.com/libsdl-org/SDL/releases/tag/release-2.0.18) indicate `SDL_RenderGeometry` was added.

My understanding is that SDL uses the odd version numbers for releases that might not be proven to be stable, so it may likely have been added in 2.0.17, but it seems users won't find much in the way of documentation for that version, so 2.0.18+ will hopefully avoid confusion.